includes/Experiments.php

```diff
@@ -219,6 +219,18 @@ public function get_experiment_groups(): array {
 	 */
 	public function get_experiments(): array {
 		return [
+			/**
+			 * Author: @brittanyirl
+			 * Issue: 8443
+			 * Creation date: 2021-07-28
+			 */
+			[
+				'name'        => 'enableAmpValidationCheck',
+				'label'       => __( 'Amp Validation Error Check', 'web-stories' ),
+				'description' => __( 'Enable AMP validation in the checklist', 'web-stories' ),
+				'group'       => 'editor',
+				'default'     => true,
```

Comment:
What's the point of adding this as an experiment if it's enabled by default from the beginning? 🤔

Reply:
my thinking was that since this is hard to trigger, it might be nice to have an off switch if something unexpected happened. probably redundant but thought it wouldn't hurt.

Reply:
meh, there's no good reason - i'm scrapping it.